src/components/ModelingSidebar/ModelingSidebar.tsx

```diff
@@ -168,7 +168,7 @@ export function ModelingSidebar({ paneOpacity }: ModelingSidebarProps) {
         height: 'auto',
       }}
       minWidth={200}
-      maxWidth={800}
+      maxWidth={2000}
```

Comment:
What about this instead? I tried 2000 locally and I was able to drag it out of the window which isn't ideal I think. But outside of that I'm super onboard!

<img width="1800" alt="image" src="https://github.com/user-attachments/assets/7525e9e2-6786-4203-9359-855caf201beb" />

```suggestion
      maxWidth={window.innerWidth - 10}
```

<img width="1800" alt="image" src="https://github.com/user-attachments/assets/e7a48cdf-3e14-4aaf-a5dc-a177acbb2a0d" />

Reply:
Sounds good.
Don't think accessing window like this in a component should effect any unit tests, if it does we can figure it out.

Reply:
Yeah you're right tho it might be good to check window first and have a fallback value?